runtime/calamari/src/lib.rs

```diff
@@ -257,15 +258,29 @@ impl Contains<Call> for BaseFilter {
             | Call::CalamariVesting(_)
             | Call::Session(_) // User must be able to set their session key when applying for a collator
             | Call::AuthorInherent(pallet_author_inherent::Call::kick_off_authorship_validation {..}) // executes unsigned on every block
-            | Call::CollatorSelection(
-                manta_collator_selection::Call::set_invulnerables{..}
-                | manta_collator_selection::Call::set_desired_candidates{..}
-                | manta_collator_selection::Call::set_candidacy_bond{..}
-                | manta_collator_selection::Call::set_eviction_baseline{..}
-                | manta_collator_selection::Call::set_eviction_tolerance{..}
-                | manta_collator_selection::Call::register_candidate{..}
-                | manta_collator_selection::Call::remove_collator{..}
-                | manta_collator_selection::Call::leave_intent{..})
+            | Call::ParachainStaking(
+                // Collator extrinsics
+                pallet_parachain_staking::Call::join_candidates{..}
```

Comment:
Can you write a comment why the implicitly disallowed extrisnics are disallowed.
Also let's just explicitly disallow them, not sure why we made this change if we don't use it.

Reply:
right now there is no non-root extrinsic in the pallet that we explicitly disallow.

We needed that with the `manta_collator_selection::register_as_candidate` extrinsic, but with this change, all `manta_collator_selection` extrinsics will be filtered so that's no longer there

Reply:
Not talking about the manta_collator_selection extrinsics but the parachain_staking ones.

Reply:
yeah. i'm saying there are no explicitly disallowed extrinsics here

Reply:
I really don't understand why we needed it before and not now :D

Reply:
> We needed that with the manta_collator_selection::register_as_candidate extrinsic, but with this change, all manta_collator_selection extrinsics will be filtered so that's no longer there

There was a user-facing non-root extrinsic that we wanted to prevent users from using. That's why I added a DISALLOWED section so we can put that extrinsic there along with reasoning on *why* it's being disallowed

Reply:
This is unnecessarily complex and just confusing. Anyway no need to fix it here.

Reply:
But please add a comment why the filtered calls are filtered.

Reply:
done